docs/src/community/phrasebook.rst

```diff
@@ -0,0 +1,67 @@
+.. include:: ../common_links.inc
+
+.. _phrasebook:
+
+Package Phrasebook
+===================
+
+There are a number of similar packages to Iris, and a lot of these have their own
+terminology for similar things. Whether you're coming or going, we hope this might
+be a helpful guide to these differences!
+Definitions for each can be found in :ref:`glossary`. See also
+`Xarray terminology <https://docs.xarray.dev/en/stable/user-guide/terminology.html>`_.
+
+.. list-table:: Phrasebook
+   :widths: 25 25 25 50
+   :header-rows: 1
+
+   * - Iris
+     - Xarray
+     - Example
+     - Notes
+   * - Non-Lazy
+     - Eager
+     -
+     - Used to relate to functions, rather than the data.
+   * - Cube
+     - DataArray
+     -
+     -
+   * - CubeList
+     - Dataset
+     -
+     - Though similar, a CubeList is a simpler object, and is
+       not a perfect comparison to a Dataset
+   * - Merge/ Concatenate
+     - Concatenate
+     - `Xarray concatenate <https://docs.xarray.dev/en/stable/user-guide/combining.html#concatenate>`_
+     - Xarray's concatenate has the capability to largely do what both
+       iris merge and iris concatenate do, this is not a perfect comparison,
+       please see the link for more information.
+   * -
+     - Merge
+     - `Xarray merge <https://docs.xarray.dev/en/stable/user-guide/combining.html#merge>`_
+     - Xarray's Merge function doesn't map neatly map to any iris feature.
+       Please see the link for more information.
+   * -
+     - Scalar
+     -
+     - Iris makes a distinction between scalar coordinates and non-scalar
+       coordinates, xarray does not. In xarray, a scalar is any non-array or
+       array-like object e.g. int, float or str.
+   * - AuxCoord
+     - Non-Dimensional Coordinate
+     -
+     - Coordinates in Iris and Xarray are categorised using rules, and so are not a
+       one-to-one match. This comparison is most accurate, but others could
+       also work.
+   * - DimCoord
+     - Dimension Coordinate
+     -
+     - Coordinates in Iris and Xarray are categorised using rules, and so are not a
+       one-to-one match. This comparison is most accurate, but others could
+       also work.
```

Comment:
I don't think this last sentence is necessary and I think this would be clearer without it.